x/farming/keeper/proposal_handler.go

```diff
@@ -103,8 +105,13 @@ func (k Keeper) UpdatePublicPlanProposal(ctx sdk.Context, proposals []*types.Upd
 			return sdkerrors.Wrapf(sdkerrors.ErrNotFound, "plan %d is not found", p.GetPlanId())
 		}
 
-		switch plan := plan.(type) {
-		case *types.FixedAmountPlan:
+		if p.EpochAmount.IsAllPositive() {
+			if p.GetName() != "" {
+				if err := plan.SetName(p.GetName()); err != nil {
```

Comment:
currently `SetName` always return `nil`, so how about using `plan.Name = p.GetName()` for the coverage?

Reply:
As discussed offline, I'll make the changes if I can't find a solution to disable codecov/patch.